fastlane/lanes/release.rb

```diff
@@ -0,0 +1,236 @@
+# frozen_string_literal: true
+
+# Lanes related to the Release Process (Code Freeze, Betas, Final Build, AppStore Submission…)
+#
+platform :ios do
+  # Executes the initial steps needed during code freeze
```

Comment:
What do you think of 

```suggestion
  # Executes the initial steps of the code freeze
```

or

```suggestion
  # Executes the initial code freeze steps
```

?